crates/release_plz/tests/all/release_pr.rs

```diff
@@ -534,6 +534,69 @@ async fn release_plz_honors_features_always_increment_minor_flag() {
     .assert_eq(&gitea_release.body);
 }
 
+#[tokio::test]
+#[cfg(unix)]
+#[cfg_attr(not(feature = "docker-tests"), ignore)]
+async fn release_plz_detects_symlink_package_changes() {
+    let context = TestContext::new().await;
+    let readme = "README.md";
+    let readme_symlink = "README_SYMLINK.md";
+    symlink_readme(&context, readme, readme_symlink);
+    let cargo_toml_path = context.repo_dir().join(CARGO_TOML);
+    let mut cargo_toml = LocalManifest::try_new(&cargo_toml_path).unwrap();
+
+    // Exclude irrelevant files from the package to ensure changes are being tracked explicitly
```

Comment:
last comment before this pr can be merged. I don't get why we only include `src`. how is this relevant for this test? 

does the test fails if we remove this `include`? If not, why having it?

Reply:
Without explicitly setting `include` here, the original README will be included in the package since all files are included by default. This means that this test still succeeds without the changes in this PR. We have to include the symlinked version only in order for this test to verify that the path normalization is working correctly. I can try to say this more succinctly and update the comment.